### ISSUE TITLE
March 28 2023 - plugin update

### ISSUE THREAD

**Issue by curtismchale** [proudcity/wp-proudcity#2231]
## Plugin Updates

- Admin Menu Editor 1.10.4 -> 1.11
- Broken link checker still requires an update - see #2216 
- Gravity Forms 2.7.2 -> 2.7.3
- Popup Maker 1.18.0 -> 1.18.1
- SO Widgets Bundle 1.46.6 -> 1.46.7
- WP Media Folder 5.5.4 -> 5.5.6
- WP Rocket 3.12.5.3 -> 3.12.6.1
- WordPress SEO 20.1 -> 20.4

### Admin Menu Editor

Release Notes: https://en-ca.wordpress.org/plugins/admin-menu-editor/#developers

I don't see anything that should affect us.

### Gravity Forms

Release Notes: https://docs.gravityforms.com/gravityforms-change-log/

I don't see anything that should affect us.

### Popup Maker

Release Notes: https://en-ca.wordpress.org/plugins/popup-maker/#developers

There is a security fix here that we should get out.

### SO Widgets Bundle

Release Notes: https://en-ca.wordpress.org/plugins/so-widgets-bundle/#developers

Minor patch nothing that should affect us.

### WP Media Folder

Release Notes: https://www.joomunited.com/changelog/wp-media-folder-changelog

Release notes are not very descriptive. Nothing jumps out at me.

### WP Rocket

Release Notes: https://wp-rocket.me/changelog/

Some general fixes to caching that might help clean up some of the caching issues we've been having.

### WordPress SEO

Release Notes: https://en-ca.wordpress.org/plugins/wordpress-seo/#developers

I don't see anything that should affect us.

**Comment by curtismchale** [comment]
@lukefretwell @kevindherman plugins are up on beta for any testing. Overall the updates are minor so I don't expect anything to be an issue.

**Comment by kevindherman** [comment]
very quick poke around and no concerns from me. @curtismchale @lukefretwell

**Comment by curtismchale** [comment]
Pertinent to #2233 WP Media Folder 5.5.6 introduced a new function filtering on `the_content`. This function called `wpmfFindImages` used `attachment_url_to_postid()` which is a standard WordPress function that is known to be not super performant.

We saw ELGL have a big spike in database usage which makes sense since we were now searching through 500k rows on each page load to see if there was an image we needed to do something with. 

We removed this function in `wp_proud_core` [here](https://github.com/proudcity/wp-proud-core/commit/06ad84202b55c796b66ea109a1f0313d4ee0b93a).

[This version of composer.json](https://github.com/proudcity/wp-proudcity/commit/46e33b477c747dea6b986ff4424b9533b55493d1) updates the plugin and contains the fix to remove the function above from `wp-media-folder`.

The build is: `BUILD=staging-1c47cff4ede319279860f1eee4b9ba05e51f667b`
